.github/workflows/check_build.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 env:
   AWS_REGION: us-east-1
-  AWS_INSTANCE_TYPE: c5a.8xlarge  # 32 vcpu, 64gb ram, $1.392 hourly
+  AWS_INSTANCE_TYPE: c5.4xlarge # 16 vcpu, 32gb ram, $0.68 hourly
```

Comment:
please resolve the warnings, but maybe just get rid of the comment alltogether.
prices can change and this might become wrong in the future. I've been trying to get rid of these.